Source/Atomic/UI/UI.cpp

```diff
@@ -535,11 +539,52 @@ void UI::HandleUpdate(StringHash eventType, VariantMap& eventData)
         exitRequested_ = false;
         return;
     }
+    
+    mouseStayedTime += eventData[Update::P_TIMESTEP].GetFloat();
```

Comment:
This could be implemented as a TB custom event, based on hover time: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/ThirdParty/TurboBadger/tb_inline_select.cpp#L160